rfcs/text/0030/process.yml

```diff
@@ -63,6 +63,38 @@
 
         Note: A non-interactive process can belong to an interactive session and is simply one that does not have open file descriptors reading the controlling TTY on FD 0 (stdin) or writing to the controlling TTY on FD 2 (stderr). A backgrounded process is still considered interactive if stdin and stderr are connected to the controlling TTY.
 
+    - name: top_level
```

Comment:
naming fun - at first blush "top_level" doesn't mean much.  It means the process in which top_level is set (process.group_leader, process.session_leader, or process.entry_leader)  is/is-not the same as the main process this event about (ie <event-doc>.process).   It would be implicitly true for <event-doc>.process itself

so 2 things: is top_level the best name for this or is thee something clearer like  "same_process"/bool?

Is there a default value specification part of ECS that we should use to identify the default value to assume for a field if it's absent?  Ie.g. process.top_level is implicitly true but doesn't need to be explicitly given in each event.

Reply:
Naming fun for sure. Maybe **same_as_process**?     process.entry_leader.same_as_process: true vs process.entry_leader.same_process: true

I haven't seen a way to specify defaults, just an "example" value. Hopefully the description is enough to convey when this is expected to be present on the process fieldset.